index.html

```diff
@@ -251,24 +251,25 @@ <h2 class="app-for-u__title">we have an app for u</h2>
         <img src="./images/appImage.png" alt="App for u" class="app-for-u__image">
     </section>
     <section class="primlancers">
-        <h2 class="primlancers__title">Primlancers
-            <span class="primlancers__title title_black">at work</span>
+        <h2 class="heading heading_span primlancers__title">Primlancers
+            <span class="title_black">at work</span>
```

Comment:
Just a quick fix
You'll want `<h2 class="heading primpancers__title> <span class="heading_span"> Primlancers </span> at work </h2>`

The heading_span has been used for the purple coloring in the header, for just the color modification on the part of the heading where it's being applied. 

Doing this will remove your need for extra styling to color the title portion to black,

Reply:
Thank you. Fixed.